### ISSUE TITLE
Suggestion: User Automations

### ISSUE THREAD

**Issue by CCOSTAN** [balloob/home-assistant.io#49]
[This] (https://github.com/balloob/home-assistant.io/pull/46) and [this] (https://github.com/balloob/home-assistant.io/issues/45) made me think of a suggestion for the site.  An automation showcase.  Some of the hardest parts of automation are just coming up with an idea.  I would love to see a place that has other automations that people have made.  Not sure how this would be laid out or contributed but I think it would help give people a roadmap of where to take their homes in terms of automation/scheduling.  Everyone has different equipment of course but I know I would find it useful.  I think one of the key things with IFTTT is the collection of recipes that they have and can be shared. 

It's just an inkling of an idea but here are some of the things I think of when thinking about this : 

* [iOS Workflow] (http://www.workflowgallery.co/)
* [Tasker ideas] (http://tasker.wikidot.com/profile-index)
* [Yahoo Pipes] (https://pipes.yahoo.com/pipes/pipes.popular)
* [OpenHab Ideas] (https://github.com/openhab/openhab/wiki/Samples-Tricks)

Maybe the last item is the way to go (Github Wiki) so that it could be organically grown by the user community. (and probably the quickest to implement) 

Bottom line: I would love to see how other people use Home-Assistant to give me ideas on how I can extend it's use in my own home.  Due to it's name, it's super hard to google how people do it and the Mailing list and Chat aren't good places for historical information/ideas.

**Comment by balloob** [comment]
I like the OpenHab trick page, good place to have the auto start and other command line foo.

For automation showcases, the plan is to keep these kind of topics on the blog. Adding them as pages to the website means we have to keep them updated, blogs are ok to go stale.

**Comment by CCOSTAN** [comment]
> I like the OpenHab trick page, good place to have the auto start and other command line foo.

Yeah - That makes sense.

**Comment by balloob** [comment]
I am not sure if we should go the wiki route or just make a page on the website that people can add to via pull requests. Adding a wiki will create a second source of information which might actually lead to more confusion in the long run.

**Comment by CCOSTAN** [comment]
I see benefits in both routes.  Wikis are easy for people to use and update.  The Pull Requests can be intimidating for new users to github (I was/is one) BUT a simple single page that is easily changed (maybe with a clear document on the steps to `Show us your project`) could be a great way to get more people (non coders) to contribute to the website resulting in richer documentation.

**Comment by rmkraus** [comment]
For what it is worth, I also vote wiki. Paulus, I can stand one up for you
and let you play around with it if you're interested. This might also
require migrating the github pages though...

On Thursday, September 10, 2015, Carlo Costanzo <notifications@github.com>
wrote:

> I see benefits in both routes. Wikis are easy for people to use and
> update. The Pull Requests can be intimidating for new users to github (I
> was/is one) BUT a simple single page that is easily changed (maybe with a
> clear document on the steps to Show us your project) could be a great way
> to get more people (non coders) to contribute to the website resulting in
> richer documentation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant.io/issues/49#issuecomment-139359811>
> .
>


-- 
Thanks,

Ryan Kraus

**Comment by CCOSTAN** [comment]
I think another advantage to the wiki is you don't have to maintain any look or feel for it consistent with the overall Home-Assistant.io site.  It can look different since it is 'User Generated'.

Depending on user contributions, it might be hard to maintain a consistent look and feel with Home-Assistant.io.

**Comment by balloob** [comment]
We can setup a subdomain wiki.home-assistant.io that points at a different
host then main site. Wouldn't a forum be easier then? Subforum for
tutorial, installation issues, development, feature requests

**Comment by CCOSTAN** [comment]
If a forum is an option on the table, I would vote for a forum.  Historical information, searching, discussion.  I like all those things. :)

**Comment by rmkraus** [comment]
Forum would be nice too. I can throw one of those up too. Does your ssl
cert cover wild cards?

On Thursday, September 10, 2015, Carlo Costanzo <notifications@github.com>
wrote:

> If a forum is an option on the table, I would vote for a forum. Historical
> information, searching, discussion. I like all those things. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant.io/issues/49#issuecomment-139422544>
> .
>


-- 
Thanks,

Ryan Kraus

**Comment by balloob** [comment]
I don't have my own ssl cert but use cloudflare free tier

**Comment by rmkraus** [comment]
I have a suspicion that a forum would not be able to use this. Either way,
let me know if you want something setup. I can deploy pretty quickly.

On Friday, September 11, 2015, Paulus Schoutsen <notifications@github.com>
wrote:

> I don't have my own ssl cert but use cloudflare free tier
>
> On Fri, Sep 11, 2015, 04:37 Ryan Kraus <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
> > Forum would be nice too. I can throw one of those up too. Does your ssl
> > cert cover wild cards?
> >
> > On Thursday, September 10, 2015, Carlo Costanzo <
> notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> > wrote:
> >
> > > If a forum is an option on the table, I would vote for a forum.
> > Historical
> > > information, searching, discussion. I like all those things. :)
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> >
> https://github.com/balloob/home-assistant.io/issues/49#issuecomment-139422544
> > >
> > > .
> > >
> >
> >
> > --
> > Thanks,
> >
> > Ryan Kraus
> >
> > —
> >
> >
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/balloob/home-assistant.io/issues/49#issuecomment-139521369
> >
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant.io/issues/49#issuecomment-139529648>
> .
>


-- 
Thanks,

Ryan Kraus

**Comment by balloob** [comment]
Closing this issue as we have the [cookbook](https://home-assistant.io/cookbook/) on the website and a [forum](https://automic.us/forum/) to share automations.
